src/utils/api.ts

```diff
@@ -25,27 +25,25 @@ export class ContentWrapper {
     });
   }
 
-  async get<T>(entity: string, options: any = {}): Promise<T[]> {
+  async get(entity: string, options: any = {}): Promise<any[]> {
     if (this.client === undefined) {
       await this.build();
     }
 
     const [entries, schema] = await Promise.all([
-      this.client.getEntries({
+      this.client!.getEntries({
```

Comment:
Looking back at this, instead of doing the `!` assertion it would be better for `this.build` to return `createClient(...)` and change the beginning lines of this function to:

```
if (this.client === undefined) {
      this.client = await this.build();
}
```

From there, TypeScript will be able to figure out that `this.client` exists without the `!` assertion.

Reply:
I'll go with this.